tasks/messages/receipt/task.go

```diff
@@ -81,10 +81,11 @@ func (t *Task) ProcessTipSets(ctx context.Context, current *types.TipSet, execut
 				Height:    int64(current.Height()),
 				StateRoot: current.ParentState().String(),
 
-				Message:  msg.Cid().String(),
-				Idx:      index,
-				ExitCode: int64(rec.ExitCode),
-				GasUsed:  rec.GasUsed,
+				Message:    msg.Cid().String(),
+				Idx:        index,
+				ExitCode:   int64(rec.ExitCode),
+				GasUsed:    rec.GasUsed,
+				EventsRoot: rec.EventsRoot.String(),
```

Comment:
I don't think we need to record the individual events for each VM message receipt, nor am I sure that VMMessage receipts even contain events. Let's drop this change since the on-chain receipt will already contain all of this information.

Reply:
I asked on #fil-lotus-dev and the consensus seems to be that the VM Message receipt will have the event root https://filecoinproject.slack.com/archives/CP50PPW2X/p1675129261393459 (or are they talking about FVM?). prob stupid question but how does a regular message receipt contain info about a vm message receipt?